docs/sdk/wrapped-keys/sign-message.md

```diff
@@ -0,0 +1,241 @@
+
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork } from "@lit-protocol/constants";
+
+const litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.Cayenne,
+    debug: false,
+});
+await litNodeClient.connect();
+```
+
+### Generating PKP Session Signatures
+
+The `LIT_PKP_PUBLIC_KEY` environment variable is required. This PKP should be owned by the corresponding Ethereum address for the `ETHEREUM_PRIVATE_KEY` environment variable.
+
+The PKP's Ethereum address will be used for the Access Control Conditions used to encrypt the generated private key, and by default, will be the only entity able to authorize decryption of the private key.
+
+```ts
+import { EthWalletProvider } from "@lit-protocol/lit-auth-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from "@lit-protocol/auth-helpers";
+
+const pkpSessionSigs = await litNodeClient.getPkpSessionSigs({
+    pkpPublicKey: process.env.LIT_PKP_PUBLIC_KEY,
+    authMethods: [
+        await EthWalletProvider.authenticate({
+            signer: ethersSigner,
+            litNodeClient,
+            expiration: new Date(Date.now() + 1000 * 60 * 10).toISOString(), // 10 minutes
+        }),
+    ],
+    resourceAbilityRequests: [
+    {
+        resource: new LitActionResource("*"),
+        ability: LitAbility.LitActionExecution,
+    },
+    ],
+    expiration: new Date(Date.now() + 1000 * 60 * 10).toISOString(), // 10 minutes
+});
+```
+
+When `getPkpSessionSigs` is called, the following happens:
+
+1. Session Keys are generated locally
+2. A request to the Lit network is submitted to authorize our Session Keys to perform the requested resource abilities (`resourceAbilityRequests`)
+3. The Lit network executes our `authMethods` to authenticate us
+   - In our case we're using `EthWalletProvider.authenticate` to generate an Authentication Signature (a Sign in With Ethereum message) that verifies we have access to the private key of a corresponding Ethereum address.
+4. The Lit network checks whether or not the authenticated Ethereum address is authorized to use the PKP
+5. If authorized, the Lit network produces signature shares for our PKP, signing a message that grants our Session Keys with the requested resource abilities
+6. The PKP signature shares are returned to us to use for our `generatePrivateKey` request
+
+### Signing a Message With A Wrapped Key
+
+Now that we have all that we need, we can call `signMessageWithEncryptedKey` to sign a message with a Wrapped Key:
+
+<Tabs
+defaultValue="evm"
+values={[
+{label: 'Signing for an EVM Based Network', value: 'evm'},
+{label: 'Signing for Solana', value: 'solana'},
+]}>
+<TabItem value="evm">
+
+```ts
+import { api } from "@lit-protocol/wrapped-keys";
+
+const { importPrivateKey } = api;
+
+const signature = await signMessageWithEncryptedKey({
+  pkpSessionSigs,
+  network: 'evm',
+  messageToSign: "The answer to the Universe is 42.",
```

Comment:
How about this? We need to factually correct
```suggestion
  messageToSign: "The answer to the Universe is 420.",
```

Reply:
[The answer to life, universe and everything](https://youtu.be/aboZctrHfK8?si=axO8EHh-eVtGY1E_)